nix/shell.sh

```diff
@@ -225,15 +225,29 @@ function apache2_start(){
 }
 
 function rust_integration(){
+    echo "rust_integration: Cleaning previous build"
     rm -rf build/
+    echo "rust_integration: Configuring with CMake (RelWithDebInfo, intern libcrypto)"
     cmake -B build . \
```

Comment:
Can we break this up to match the current helper command flow (`configure;build;test`) ?  I realize you're changing some currently hard coded flags, e.g.: `BUILD_TESTING` and interning, but this all in one function means we can't track timing metrics for each step.